ledger/internal/eval.go

```diff
@@ -927,7 +927,7 @@ func (eval *BlockEvaluator) Transaction(txn transactions.SignedTxn, ad transacti
 // TransactionGroup tentatively adds a new transaction group as part of this block evaluation.
 // If the transaction group cannot be added to the block without violating some constraints,
 // an error is returned and the block evaluator state is unchanged.
-func (eval *BlockEvaluator) TransactionGroup(txgroup []transactions.SignedTxnWithAD) error {
+func (eval *BlockEvaluator) TransactionGroup(txgroup []transactions.SignedTxnWithAD) (returnErr error) {
```

Comment:
nit: Seems awkward to have the `returnErr` name, rather than calling it `err` and removing the declaration of err where it first appears below.

(Same going for the similar thing in `eval.go`)